### PR TITLE
Preliminary format flags support

### PR DESCRIPTION
**Pull request by carsonfarmer**
This updates to the latest SDK release, and changes the format flag values slightly to match the new incoming validator query format flags. This can be considered a patch update, because while the format flags have changed slightly, the defaults have also been updated to reflect the current settings. So using defaults still gets you the same behavior. The new options are now: "pretty" (table output), "table" (existing default with rows and columns), and "objects" (future default, with array of objects). We do not yet support `unwrap` or `extract`, but these options will come in the next major release.

### CODE REVIEW

File: src/commands/read.ts [hunk 1]
```diff
@@ -28,8 +28,8 @@ export const builder: CommandBuilder<Options, Options> = (_yargs) =>
     })
     .option("format", {
       type: "string",
-      description: "Output format. One of 'raw', 'tabular', or 'objects'.",
-      default: "raw",
+      description: "Output format. One of 'pretty', 'table', or 'objects'.",
+      default: "table",
```
Comment (carsonfarmer) [inline src/commands/read.ts:32]: Updates the default to match the current settings (which I had called "raw").

File: src/commands/read.ts [hunk 2]
```diff
@@ -45,10 +45,11 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
...
   };
   try {
-    const res = await connect(options).read(query);
-    const formatted = format === "raw" ? res : resultsToObjects(res);
+    const res = await connect(options).read(query); // Defaults to "table"
+    // After we upgrade the SDK to version 4.x, we can drop some of this formatting code
+    const formatted = format === "table" ? res : resultsToObjects(res);
 
-    if (format.startsWith("tab")) {
+    if (format === "pretty") {
```
Comment (carsonfarmer) [inline src/commands/read.ts:52]: Now tabular is renamed to "pretty"... naming things is hard!

File: src/commands/read.ts [hunk 3]
```diff
@@ -45,10 +45,11 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     chain,
   };
   try {
-    const res = await connect(options).read(query);
-    const formatted = format === "raw" ? res : resultsToObjects(res);
+    const res = await connect(options).read(query); // Defaults to "table"
```
Comment (carsonfarmer) [inline src/commands/read.ts:48]: We're taking advantage of the fact that the SDK 3.1.2 release is smoothing things over for us for now.

**Review by carsonfarmer — COMMENTED** [review]
Hopefully a pretty simple and easy review for you @joewagner?

**Review by joewagner — APPROVED** [review]
Looks great! Works great!  I did a create, write and read.  Also created a token locally.

File: src/commands/read.ts (hunk 2 above)
Comment (joewagner) [reply src/commands/read.ts:52]: Love it!

**Pull request merged**